std/zip.d

```diff
@@ -1,7 +1,7 @@
 // Written in the D programming language.
 
 /**
- * Read/write data in the $(LINK2 http://www.info-_zip.org, zip archive) format.
+ * Read/write data in the $(LINK2 http://www.info-zip.org, zip archive) format.
```

Comment:
Pull request to tackle this in the macro definitions: https://github.com/dlang/dlang.org/pull/1338

Should still put the underscore before the second "zip", though.